docs/modules/ROOT/pages/presets.adoc

```diff
@@ -44,3 +44,38 @@ NOTE: Class hashes were computed using {class-hash-cairo-version}.
 |===
 
 TIP: {starkli} class-hash command can be used to compute the class hash from a Sierra artifact.
+
+== Presets Usage
+
+:setup_project: xref:index.adoc#set_up_your_project[setting up a project]
+:install_lib: xref:index.adoc#install the_library[installing the Contracts for Cairo library]
+:presets_dir: link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.10.0/src/presets[presets directory]
+
+These preset contracts are ready-to-deploy which means they should already be declared on the Sepolia network.
+Simply deploy the preset class hash and add the appropriate constructor arguments.
+Deploying the ERC20 preset with {starkli}, for example, will look like this:
+
+[,bash, subs="verbatim,attributes"]
+----
+starkli deploy {ERC20-class-hash} \
+  <CONSTRUCTOR_ARGS> \
+  --network="sepolia"
+----
+
+If a class hash has yet to be declared, copy/paste the preset contract code and declare it locally.
```

Comment:
should we add more detail/an example of how copypasting should be done?
can we do something better than copypasting? distributing built artifacts?

Reply:
My two cents fwiw, copypasting is consistent with the expected flow in Wizard, and distributing built artifacts for a thing you declare just once seems like an overhead, more when we favor wizard long term instead of local presets.

Reply:
I take the point, but i still think copypasting can be error prone and it doesn't feel quite right for it to be the recommended way of using code. even wizard offers downloading the files which imo is way better.

as of the overhead, we could offer artifacts as a submodule/separate module, so i don't think it'd be that much of a burden if properly automated.

i guess we can iterate it over time, from copypasting to something a bit more reliable.

Reply:
> I take the point, but i still think copypasting can be error prone and it doesn't feel quite right for it to be the recommended way of using code.

Since the class hashes should be declared (as part of the new release process), users shouldn't have to do any of this. I do get the apprehension for c/p though. I'll add an example

> as of the overhead, we could offer artifacts as a submodule/separate module, so i don't think it'd be that much of a burden if properly automated.
i guess we can iterate it over time, from copypasting to something a bit more reliable.

+1